tests/Unit/GroupTest.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use App\Judite\Models\Group;
+use App\Judite\Models\Course;
+use App\Judite\Models\Student;
+use App\Judite\Models\Invitation;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class GroupTest extends TestCase
```

Comment:
Unit tests should be CamelCase and should test the model's methods functionalities, and for that don't need to make any http requests, that is for the feature tests...

Reply:
👍